CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modified `skbio.stats.composition.ilr` and `skbio.stats.composition.ilr_inv` to suppress errors when inputing a 1D basis.  ([#1393](https://github.com/biocore/scikit-bio/issues/1393)).  Also fixing how the basis is handled in `skbio.stats.composition.ilr_inv`.
```

Comment:
Alrighty.  Here's a run down of what happened.

- `_check_orthogonality` only accepts a basis that is within the simplex.  The basis passed into `ilr_inv` lives only in `ilr` space, so in order to make sure that the basis is correct (i.e. orthogonal), it needs to be transformed into the simplex space to match `_check_orthogonality`
- So basically, you need to be consistent between these two spaces (`ilr` space, and simplex space).  And I just fixed this mistake to enforce this consistency.  I didn't catch this before, because we didn't have a test case for passing in a basis for `ilr_inv`.

We now have that fix.  Let me know if there is anything else that I need to clarify.